docs/protocol.md

```diff
@@ -61,7 +63,11 @@ enums used.
   'size'                   =>  8,
   'message_counter'        =>  9,
   'correlation_id'         => 10,
-  'complete_message_count' => 11
+  'complete_message_count' => 11,
```

Comment:
i feel separating enums based on type of event is good idea, rather than having 1 enum.